### ISSUE TITLE
Add debug support for MSVC builds

### ISSUE THREAD

**Issue by jayvdb** [astral-sh/python-build-standalone#26]
I dont see `--debug` support in the Windows build scripts.  Debug builds are supported by the CPython PCbuild system. https://github.com/python/cpython/blob/master/PCbuild/readme.txt

**Comment by indygreg** [comment]
Yes, this would be nice. The hacks to the Windows build system are ugly and the lack of debug support is mainly a side-effect of me wanting to preserve more of my sanity by not having to hack on the Visual Studio builds any more than necessary.

**Comment by vadikmironov** [comment]
Windows debug builds are now published, starting with release 20260924 (https://github.com/astral-sh/python-build-standalone/releases/tag/20260924): debug for 3.10+ on i686 and x86_64 and 3.11+ on aarch64, plus freethreaded+debug for 3.13+ on all three (-full archives only).

@jjhelmus, thanks a lot for #1283 and for reviewing and merging #1262 / #1287! I think the issue can be closed now, could you close it when you get a chance?

**Comment by jjhelmus** [comment]
Thanks for all your work on getting this created @vadikmironov  🎆.
